project/build.properties

```diff
@@ -1 +0,0 @@
-sbt.version = 0.13.12
```

Comment:
Do you have any particular problem with the fixed sbt version, or is it just a general intention? I'm okay with the change; just curious.